### ISSUE TITLE
Downloading the pywbem os_setup scripts

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#1744]
The documents define getting these scripts as follows in the installation section

Download the pywbem_os_setup.sh script from one of these sources:

    pywbem_os_setup.sh on this site
    pywbem_os_setup.sh on the master branch of the pywbem Git repository

Originally these were defined at a stable url in the readthedocs web site.  However, now the url for the .sh  file in read-the-docs is something like:

https://pywbem.readthedocs.io/en/stable/_downloads/fabe1e4046cdd77354c977796ba8645b/pywbem_os_setup.sh

The inclusion of the unique id would indicate that this is not too stable and is certainly not easy to include in any automation (i.e. a make file the automatically gets the file).  Do we know why this happened?  Should we change the text in the documentation.

Note also that it changed from http to https.

**Comment by KSchopmeyer** [comment]
We recognize that we do have an issue with putting the url for this script into tools that automate the installation of packages that use pywbem because 1. the RTD definition changes, 2. the user must account for pywbem versioning if the github version is used.

We suggest that we will look at the setupotools extras capabilities to see if that is a way the scripts can be included in the PIP files so that it can be used as part of the installation of Pywbem.

**Comment by KSchopmeyer** [comment]
Andy suggested that we download the appropriate os_setup script based on pywbem version. We cannot do that in general and would have to fix the pywbem version used in the application requirements documentation since you cannot know the pywbem version until you have piped it and you need the os setup script before you pip install pywbem.

**Comment by andy-maier** [comment]
The hex string in the URL for the pywbem_os_setup.sh script on the RTD site is generated by the site itself, as a result of processing the RST source code line:
```
:download:`pywbem_os_setup.sh <../pywbem_os_setup.sh>`
```
Apparently, the RTD site changed the algorithm by which the URL is created, so that since recently, it contains a hex string.

The presence of the hex string prevents automating the download of the script by packages that depend on pywbem, because the value of the hex string is (probably) unpredictable.

One way of having a predictable download URL for the script, for a given pywbem version, would be to use:
```
https://raw.githubusercontent.com/pywbem/pywbem/stable_{m.n}/pywbem_os_setup.sh
```
where `{m.n}` is e.g. `0.14`.

However, there is no such URL currently that automatically resolves to the latest version released to Pypi. In theory, we could create a branch for the latest released version, and carry that branch forward any time we release a version to Pypi. However, that would make the release process even more complex.

**Proposal:**

Packages that depend on the latest version of pywbem released to Pypi can use the master version of the script using the URL that is already shown in the pywbem installation section. That is close enough to the latest released version. Creating a branch corresponding to the latest released version does not seem worth the effort.

The installation section would need to be adjusted a little to explain that.

**Discussion point:** Discuss the proposal.

**Comment by andy-maier** [comment]
If I recall correctly, we discussed this issue and agreed on the proposed approach. I have created PR #1762 that implements the proposal.

**Comment by andy-maier** [comment]
PR #1764 rolls the fix back into 0.14.4
